tests/test_user_document.py

```diff
@@ -54,3 +55,27 @@ def test_post_patient_report(self):
         with open(fpath, 'r') as udoc_file:
             self.assertEqual(udoc_file.read(),test_contents)
         os.remove(fpath)
+
+
+    def test_download_user_document(self):
+        service_user = self.add_service_user()
```

Comment:
This will work - but there's nothing about your API that requires a "service user", and I think it'll just introduce complexity.

Do you mind altering the test to just use the built in test_user?  i.e. self.login() w/ the default user_id of TEST_USER_ID, which can also be used in the format calls below?